lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -866,6 +896,31 @@ def test_pagination(self):
             }
         )
 
+    @ddt.data(
+        (True, "endorsed_comment"),
+        ("true", "endorsed_comment"),
+        ("1", "endorsed_comment"),
+        (False, "non_endorsed_comment"),
+        ("false", "non_endorsed_comment"),
+        ("0", "non_endorsed_comment"),
+    )
```

Comment:
Test case for something other than we expected? E.G. "Balloon", "Fasle", "fAlSe?